### ISSUE TITLE
Is this plugin too risky to use due to lack of support?

### ISSUE THREAD

**Issue by AndrWeisR** [Crypho/cordova-plugin-secure-storage#140]
As the readme states:

> As the company behind SecureStorage is moving away from Cordova, we can allocate very little time to further develop SecureStorage. If you are interested in taking over the maintenance and development of the plugin please contact us at dev@crypho.com.

Does that make this plugin too risky to use for a production app? What happens if it breaks and your access to native secure storage disappears?

**Comment by zielinskikamil** [comment]
Good question +1

**Comment by DennisSmolek** [comment]
That is what has happened with [The Intel Security Plugin](https://github.com/AppSecurityApi/com-intel-security-cordova-plugin)

My suggestion is either to use this repo and hope that it keeps getting (somewhat) maintained right up until the point it breaks for you but make testing it a priority for your update/feature builds so you can detect it early.. 

At that point, if no one has taken this over you can fork it and work on fixing the flaw or hiring folks to fix it for you with this repo as a starting point. 

The Intel platform hasn't been touched in 2 years, this one is still getting issues and PR requests answered and pulled, and has a larger group of dev's talking about it. 

There's even an open thread about new maintainers. 

Unless you want to build native or start from scratch I don't see many other plugins/options out there unfortunately..

**Comment by ggozad** [comment]
I wasn't gonna step in but hey :)
The plugin does what it promises well and has been stable as is for a long time.
We have no plans of further expanding it because it does well what it promises.
Both I as well as @demetris-manikas have been responsive and will continue to do so as time permits even if we do not use the plugin in production any longer.

I was hoping that given the plugins inclusion in ionic core, ionic developers would step in and take over maintenance. This has not been the case and we are still waiting for devs with proven contribution and understanding of the plugin to step in. As things are I am happy to keep ownership of the package and wait for as long as it takes :)

The goals of the plugin are achieved as it is. We will not make any changes that break the API and will accept PRs that either fix any future problems or improve things without breaking anything.

TL;DR You should be fine.

**Comment by demetris-manikas** [comment]
I want to reassure users that I 'll continue to support the plugin.
I support it without being a member of the company behind it for a long time now 
and I 'll continue to do so.
Cheers.
